### DISCUSSION TITLE
What compatibility is guaranteed for future Pekko versions?

### DISCUSSION THREAD

**Discussion by erikvanoosten** [apache/pekko#581] [General]
What binary/source compatibility will be guaranteed for future Pekko versions? Is it like Akka where new versions are always binary and source backward compatible with all previous versions, and source and binary forward compatible within the same minor version (if I got that right in the first place)?

**Comment by pjfanning** [comment]
So far, we are sticking to the Akka rules.

https://pekko.apache.org/docs/pekko/current/common/binary-compatibility-rules.html
